backend/helpers/emailTemplates.json

```diff
@@ -1,6 +1,6 @@
 {
     "INVITE_ADMIN" : {
       "SUBJECT" : "Invitation for becoming an Admin",
-      "BODY" : "You are invited to join HITK Tech Community as an admin.\n\nUse {link} to sign up"
+      "BODY" : "Hey {name}! \n You are invited to join HITK Tech Community as an admin.\n\nUse {link} to sign up"
```

Comment:
The mail cannot access the variables. You didn't pass them only.
![preview](https://user-images.githubusercontent.com/36669463/106362171-65ff7380-6347-11eb-91d9-a6540d51c18b.png)
This is not the email what we expect to send